app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post, counter_cache: :like_counter
```

Comment:
- [ Required ] I think it would be better if you follow the project requirements when updating the `like_counter` field I know that you can update the `like_counter` field using `counter_cache` but according to the project requirements, this field should be updated using the `update_like_counter` method so let's follow the project guidelines. Please see the comment I've left on the `update_like_counter` method.

## Project requirements
![Screenshot from 2023-07-20 00-40-21](https://github.com/Elhambasir/blog-app/assets/16392046/058b0084-81d0-4568-bf8c-92d028596571)